manifests/init.pp

```diff
@@ -161,19 +165,13 @@
   Optional[Hash] $blackout_windows = undef,
 ) {
 
-  # None tunable
-  $cache_dir = lookup('os_patching::cache_dir',Stdlib::Absolutepath,first,undef)
-  $fact_dir = lookup('os_patching::fact_dir',Stdlib::Absolutepath,first,undef)
-  $fact_file = lookup('os_patching::fact_file',String,first,undef)
-  $fact_upload_cmd = lookup('os_patching::fact_upload_cmd',String,first,undef)
-
```

Comment:
Maybe someone can clarify the intention behind this "None tunable" since it's user-overrideable using Hiera… So I moved this above in parameters but I might be breaking the initial intention.

Reply:
There are a few hardcoded instances of where these locations should be (mainly in the patch task).  I want to be able to have them live anywhere, but right now we need them to stay where they're put.  Make sense?

Reply:
Hiera does not look like the best solution here, but I guess there is no ideal solution for this problem.  I reverted it.